src/components/PaginatedTable/useScrollBasedChunks.ts

```diff
@@ -51,10 +51,18 @@ export const useScrollBasedChunks = ({
             Math.floor(visibleEnd / rowHeight / chunkSize) + overscanCount,
             Math.max(chunksCount - 1, 0),
         );
-
         return {start, end};
     }, [parentRef, tableRef, rowHeight, chunkSize, overscanCount, chunksCount]);
 
+    React.useEffect(() => {
+        const newRange = calculateVisibleRange();
+
+        if (newRange) {
+            setStartChunk(newRange.start);
+            setEndChunk(newRange.end);
+        }
+    }, [chunksCount, calculateVisibleRange]);
+
```

Comment:
Why do you need this?

Reply:
The problem was the next: if we have a lot of date and scroll down, after that change filter so no data is presented, PaginatedTable calculated two active chunks instead of one:

https://github.com/user-attachments/assets/c17e4c2e-675f-43aa-9f2b-284dc6461e7a

Reply:
Good job!